src/Catty/ViewController/Continue&New/MaintainObject/MaintainScript/BrickCategoryOverviewController.swift

```diff
@@ -41,6 +41,9 @@ import UIKit
 
     override func viewDidLoad() {
         super.viewDidLoad()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
         setupCollectionView()
```

Comment:
It is best practice to also call `super.viewWillAppear`. According to Apple's documentation (https://developer.apple.com/documentation/uikit/uiviewcontroller/1621510-viewwillappear):

>  If you override this method, you must call super at some point in your implementation.

Reply:
@m-herold thanks for your review, I updated the PR.